### ISSUE TITLE
Chat icon only shows with fill

### ISSUE THREAD

**Issue by peterel** [nolimits4web/Framework7-Icons#4]
Hi,

The chat icon shows up correct on web but on simulator and device it always has a fill, i.e. as chat_fill. (Also, when using chats icon one gets one "bubble" with fill and one without.) Any idea how to get a chat icon without fill?

Thanks.

Peter

**Comment by nolimits4web** [comment]
Are you sure you are using latest fonts version from this repo? It should be ok here

**Comment by peterel** [comment]
@nolimits4web Thanks for quick reply. 

I did get them from this repo end of nov, beginning of dec if my memory serves me correctly. Is there some "easy" way to find out if Im using the  latest version somehow? Or would I need to go through the "re-install" again?

Note: All other icons (that Ive tested, perhaps 10 or so) works as expected except the chat(s). Note2: Chat(s) icon(s) works as expected, without fill, on web. Its when I get to simulator and device where it behaves a bit unexpected.

Thanks.

Peter

**Comment by nolimits4web** [comment]
Latest release was on in the beginning of January. Check and download it from releases tab https://github.com/nolimits4web/Framework7-Icons/releases

**Comment by peterel** [comment]
Alright, Ill give it a go and get back to you.

Since its been a few months since I did this I just want to check something. The instructions say that one should copy over the fonts folder. My current fonts folder holds all .svg icons whereas the one that I just downloaded and extracted only holds 4 "files" (.eot, .woff, .woff2 and .ttf). Should I really just remove my current fonts folder and replace it with this? I only use Framework7 fonts btw so its not that I have other fonts in that folder. 

Peter
